lib/services/alexa/request/lifecycle/update.ts

```diff
@@ -15,11 +18,22 @@ const update = async (runtime: AlexaRuntime): Promise<void> => {
     else if (repeatNumber < 100) repeat = RepeatType.DIALOG;
     storage.set(S.REPEAT, repeat);
   }
-
   turn.set(T.REQUEST, runtime.getRequest());
   variables.set(V.TIMESTAMP, Math.floor(Date.now() / 1000));
 
   await runtime.update();
+
+  // Track response on analytics system
+  if (runtime?.services?.analyticsClient) {
+    runtime.services.analyticsClient.track(
+      runtime.getVersionID(),
+      Event.INTERACT,
+      true,
+      runtime.getRequest(),
+      input.requestEnvelope.session?.sessionId,
+      runtime.getFinalState()
+    );
+  }
```

Comment:
is it needed here too? It's already being called in `response.ts`. If not needed here, also revert the `input` param added to this func

Reply:
solved